WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -189,20 +247,22 @@ extension BlazeCampaignsViewController: UITableViewDataSource, UITableViewDelega
         return cell
     }
 
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
Nit: ideally, the selection should be removed when you return the screen, at least that's how native apps do it.

Reply:
Agreed, but it's a bit tricky here since it's a modal sheet view. So I can't use viewWillAppear - guessing I'd have to use a delegate.